main/bind/APKBUILD

```diff
@@ -174,30 +182,58 @@ dev() {
 	mv "$pkgdir"/usr/bin/isc-config.sh "$subpkgdir"/usr/bin/
 }
 
-_py3() {
-	pkgdesc="A module allowing rndc commands to be sent from Python programs"
-	depends="$_py3deps"
+plugins() {
+	pkgdesc="The ISC DNS server plugins"
+	depends="$depends_plugins"
+
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/bind "$subpkgdir"/usr/lib/
+}
+
+py3_isc() {
+	pkgdesc="Python functions to support BIND utilities"
+	depends="$depends_py3_isc"
+	provides="py3-isc=${_isc_ver}-r${_isc_rel} py3-$pkgname=$pkgver-r$pkgrel"
+	replaces="py3-$pkgname"
+
 	mkdir -p "$subpkgdir"/usr/lib
 	mv "$pkgdir"/usr/lib/python3* "$subpkgdir"/usr/lib/
 }
 
-_dnssec_tools() {
-	pkgdesc="Utilities for DNSSEC keys and DNS zone files management"
-	depends="py3-$pkgname=$pkgver-r$pkgrel"
+pytnon_tools() {
+	pkgdesc="The ISC DNS tools using the py3-isc module"
+	depends="$depends_python_tools"
+
 	mkdir -p "$subpkgdir"/usr/sbin
-	mv  \
-		"$pkgdir"/usr/sbin/nsec3hash \
-		"$pkgdir"/usr/sbin/dnssec* \
-		"$subpkgdir"/usr/sbin/
+	for i in "$pkgdir"/usr/sbin/*; do
+		file "$i" | grep 'Python script' >/dev/null 2>&1 || continue
+		mv "$i" "$subpkgdir"/usr/sbin/
+	done
 }
 
+server_tools() {
+	pkgdesc="The ISC DNS authoritative server tools"
+	depends="$depends_server_tools"
+	provides="$pkgname-dnssec-tools=$pkgver-r$pkgrel"
+	replaces="$pkgname-dnssec-tools"
 
-plugins() {
-	pkgdesc="The ISC DNS server plugins"
-	depends="$depends_plugins"
+	mkdir -p "$subpkgdir"/usr/bin
+	for i in "$pkgdir"/usr/bin/*; do
+		case "${i##*/}" in
+			dig|host|mdig|nslookup) ;;
```

Comment:
`dig` `host` and `nslookup` are provided in a package named `bind-server-tools`? I have always seen those as clients or client tools.

Reply:
Those matches don't have a related command, so we skip over them.

We want `dig`, `host`, `mdig`, and `nslookup` to be in `bind-tools` as before.